app/javascript/packs/banner_editor.jsx

```diff
@@ -490,6 +512,35 @@ export default class BannerEditor extends React.Component {
       });
   }
 
+  incrementChannelIndex(){
+    let temp = this.state.channelIndex;
+    temp++;
+    this.setState({channelIndex: temp, loading: true},
+      () => {this.fetchBanner()}
+    )
+  }
+
+  decrementChannelIndex(){
+    let temp = this.state.channelIndex;
+    temp--;
+    this.setState({channelIndex: temp, loading: true},
+      () => {this.fetchBanner()}
+    )
+  }
+
+  toggleChannelMode(){
+    if(this.state.channelMode === true){
+      this.setState({channelMode: !this.state.channelMode, state: 'same'},
+      () => { this.save() }
+      )
+    }
+    else{
+      this.setState({channelMode: !this.state.channelMode},
+      () => { this.save() }
+      )
```

Comment:
Can we format this a little differently? The parenthesis at the end looks really strange

Reply:
Done ✔️